13-collabs.Rmd

```diff
@@ -5,6 +5,248 @@
 ottrpal::set_knitr_image_path()
 ```
 
-```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Learning objectives are to be able to: Use collaborations and code review to boost reproducibility. Polish code with code review through GitHub pull requests. Manage tasks of a project using GitHub Issues. Communicate with collaborators clearly on GitHub"}
 ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21a84b32106_0_68")
 ```
+
+## Components of collaborating on GitHub
+
+As noted previously, GitHub is not only great for sharing and version control but it is also great for collaborating!
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Github and git allow you to collaborate with others on the same files in a way that has a system for merging all the work together. In this diagram it shows how one set of changes labeled in yellow can be merged in with another set of changes labeled in blue. Reproducible parrot is happy and says ‘GitHub and git make it easy for us to collaborate with each other in a way that we can merge our work together!’ His parrot collaborator is also happy."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g341dc8b716d_0_6")
+```
+
+This collaboration can be done through the use of issues and pull requests!
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The GitHub collaboration workflow starts with the creation of an issue. Someone identified a problem or needed feature for the project and they can take note of this by creating an issue. This issue will be addressed by changes in a pull request which will reference this issue. "}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g341dc8b716d_0_0")
+```
+### Step 1. An problem identified/issue created
+
+[GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) are where we can note, plan, and discuss work for a project. It might first be filed by one person but then used to further describe a problem or feature request. Once the issue is scoped and defined enough it can be assigned to someone who can work on it
```

Comment:
```suggestion
[GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) are where we can note, plan, and discuss work for a project. It might first be filed by one person but then used for further discussion to define a problem or feature request. Once the issue is scoped and defined enough it can be assigned to someone to work on it.
```

Reply:
emphasizing the multiple people option for issues aspect. 

Do we want to change out "feature request" for something more analysis like?

Reply:
ok I just dropped the word feature... maybe that works?